src/python/test/test_ripsnet.py

```diff
@@ -0,0 +1,110 @@
+    input_dim = 2
+
+    model = RipsNet(phi_1, phi_2, input_dim, perm_op=perm_op)
+
+    test_input_raw = [np.array([[1.,2.],[3.,4.]])]
+
+    test_input = tf.ragged.constant([
+        [list(c) for c in list(test_input_raw[i])] for i in range(len(test_input_raw))], ragged_rank=1)
+
+    model.predict(test_input)
+
+    model.set_weights(weights_vect)
+
+
+    clean_data_test = [np.array([[ -7.04493841,   9.60285858],
+                                 [-13.14389003, -13.21854157],
+                                 [ -3.21137961,  -1.28593644]]),
+                       np.array([[ 10.40324933,  -0.80540584],
+                                 [ 16.54752459,   0.70355361],
+                                 [  6.410207  , -10.63175183],
+                                 [  2.96613799, -11.97463568]]),
+                       np.array([[ 4.85041719, -2.93820024],
+                                 [ 2.15379915, -5.39669696],
+                                 [ 5.83968556, -5.67350982],
+                                 [ 5.25955172, -6.36860269]])]
+
+    noisy_data_test = [np.array([[ -8.93311026,   1.52317533],
+                                 [-16.80344139,  -3.76871298],
+                                 [-11.58448573,  -2.76311122],
+                                 [-15.06107796,   5.05253587]]),
+                       np.array([[-3.834947  , -5.1897498 ],
+                                 [-3.51701182, -4.23539191],
+                                 [-2.68678747, -1.63902703],
+                                 [-4.65070816, -3.96363227]]),
+                       np.array([[ 4.7841113 , 19.2922069 ],
+                                 [10.5164214 ,  5.50246605],
+                                 [-9.38163622,  7.03682948]])]
+
+    tf_clean_data_test = tf.ragged.constant([
+        [list(c) for c in list(clean_data_test[i])] for i in range(len(clean_data_test))], ragged_rank=1)
+    tf_noisy_data_test = tf.ragged.constant([
+        [list(c) for c in list(noisy_data_test[i])] for i in range(len(noisy_data_test))], ragged_rank=1)
+
+    clean_prediction = np.array([[0.5736222, 0.3047213, 0.6746019, 0.49468565],
+                                 [0.4522748, 0.7521156, 0.3061385, 0.77519494],
+                                 [0.5349713, 0.49940312, 0.51753736, 0.6435147]])
+    noisy_prediction = np.array([[0.53986603, 0.33934325, 0.64809155, 0.49939266],
+                                 [0.5637899, 0.28744557, 0.67097586, 0.50240993],
+                                 [0.5689339,  0.55574864, 0.47079712, 0.68721807]])
+
+    #print(np.linalg.norm(clean_prediction - model.predict(tf_clean_data_test)))
+
+    assert(clean_prediction.shape == model.predict(tf_clean_data_test).shape)
+    assert(noisy_prediction.shape == model.predict(tf_noisy_data_test).shape)
+    assert(np.linalg.norm(clean_prediction - model.predict(tf_clean_data_test)) <= 1e-7)
+    assert(np.linalg.norm(noisy_prediction - model.predict(tf_noisy_data_test)) <= 1e-7)
```

Comment:
On OSX, this test [fails on CI](https://dev.azure.com/GUDHI/gudhi-devel/_build/results?buildId=1401&view=logs&jobId=cf796865-97b7-5cd1-be8e-6e00ce4fd8cf&j=cf796865-97b7-5cd1-be8e-6e00ce4fd8cf&t=d5062407-4aaa-5468-7238-c199647c44e2):
```
AssertionError: assert 1.1247726736396533e-07 <= 1e-07
```

Reply:
Thank you. I've added the documentation fix and increased the error margin to `1e-6`.